client/src/pages/PostSolution.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState, ChangeEvent, FormEvent } from "react";
+import { useParams } from "react-router-dom";
+import { Col, Container, Form, Row } from "react-bootstrap";
+import Button from "../components/Button.tsx";
+
+interface FormData {
```

Comment:
Why are we using interfaces instead of types ?

Reply:
done